### ISSUE TITLE
For the select item it's good to return the slot name instead of slot value

### ISSUE THREAD

**Issue by dibyadip-das** [keatontaylor/alexa-actions#19]
**The Scenario:**

- Alexa Says` "I noticed the TV was turned on, what would you like to play, Netflix, Amazon Prime Video or YouTube?"`
- I say `"Prime Video"`
- Received slot: `"ResponseNone"` because in the slots I mentioned Amazon Prime Video and not Prime Video

**Solution:**
Added Synonyms to solve this issue
![image](https://user-images.githubusercontent.com/47345141/81614147-3ac1a680-93fd-11ea-9907-f8c1a4348332.png)

Now when I say  `"Prime Video"` I was getting prime video. But I wanted to get "Amazon Prime Video which is the slot name in this case and not value.

So I made the changes to the first 2 line of  the following code and now it seems to work.
```
def handle(self, handler_input):
        # type: (HandlerInput) -> Response
        #selection  = ask_utils.get_slot_value(
        #    handler_input=handler_input, slot_name="Selections")
        
        selection = ask_utils.get_slot(handler_input, slot_name="Selections")
        selection = selection.to_dict()["resolutions"]['resolutions_per_authority'][0]["values"][0]["value"]["name"]

        global home_assistant_object
        if home_assistant_object == None:
            home_assistant_object = HomeAssistant(handler_input)
            home_assistant_object.get_ha_state()
            

        home_assistant_object.post_ha_event(selection)
        speak_output = "You selected " + selection
        return (
            handler_input.response_builder
                .speak(speak_output)
                # .ask("add a reprompt if you want to keep the session open for the user to respond")
                .response
        )
```
Let me know your thoughts on this ?

**Comment by keatontaylor** [comment]
Looking into this further it is kinda odd that amazon doesn't actually return the slot value when it recognizes a Synonym and there is a small issue where your code if Amazon decides for whatever reason to have the resolutions_per_authority be a list greater than a single element, because it isn't a guarantee that the first element will be the true matched resolution.

I'll need to give this one some more thought on how it can be resolved.

**Comment by dibyadip-das** [comment]
> Looking into this further it is kinda odd that amazon doesn't actually return the slot value when it recognizes a Synonym and there is a small issue where your code if Amazon decides for whatever reason to have the resolutions_per_authority be a list greater than a single element, because it isn't a guarantee that the first element will be the true matched resolution.
> 
> I'll need to give this one some more thought on how it can be resolved.

I absolutely agree with you, I wish there was a way through the SDK to retrieve the name or id. Value will make it difficult. We will need to handle a lot more conditions on the Home Assistant side.

**Comment by keatontaylor** [comment]
Resolved in 0.5 release
